### ISSUE TITLE
vim.log.level.DEBUG hides notifications instead of styling them

### ISSUE THREAD

**Issue by MichaelPetersen22** [rcarriga/nvim-notify#221]
Presumably a fault on my part, but I cannot find documented anywhere the actual USAGE of the vim.log.levels, The only documentation related to them is an description of all the ENUMS that can be used.

I am attempting to create a notification that will show with the debug 'flavouring' in that it utilizes the debug icon and debug highlights in the notification displayed.

However, On setting vim.log.level.DEBUG or alternatively 'debug' as the second parameter in require('notify')(myMsg, vim.log.level.DEBUG) the notifications simply don't show up? Is this expected behaviour? What else do I need to set to have debug level notifications showing up?

**Comment by rcarriga** [comment]
See the `level` field in `:h notify.Config`

**Comment by Normanras** [comment]
@rcarriga I'm experiencing something similar to Michael. I went through all the log levels and found the following:

Shows nvim-notify message:
* ERROR
* OFF
* WARN
* INFO

Does not show nvim-notify message:
* DEBUG
* TRACE

Additionally, I went through `:h notify.Config` and read through all the `level` information and didn't (or couldn't) find any information as to why certain log levels show messages and why others don't. [Neovim's docs](https://neovim.io/doc/user/lua.html) say the following: 
```
vim.log.levelsLog levels are one of the values defined in vim.log.levels:
vim.log.levels.DEBUG vim.log.levels.ERROR vim.log.levels.INFO vim.log.levels.TRACE vim.log.levels.WARN vim.log.levels.OFF
```

Am I missing something in your docs? Here are some repeatable commands that others can try:
* `:lua vim.notify("Test", vim.log.levels.TRACE, {})` --> Doesn't show a notification
* `:lua vim.notify("Test", vim.log.levels.ERROR, {})` --> Shows a notification

(You obviously don't need the full `vim.log.levels`, just being thorough.)

Let me know if I'm missing something obvious!
